components/aws/sagemaker/common/_utils.py

```diff
@@ -581,6 +592,208 @@ def get_best_training_job_and_hyperparameters(client, hpo_job_name):
+        else:
+            logging.error("Must specify pre-human task lambda arn and annotation consolidation post-human task lambda arn.")
+    else:
+        logging.error("Task type must be Bounding Box, Image Classification, Semantic Segmentation, Text Classification, or Custom.")
+
+    request['HumanTaskConfig']['UiConfig']['UiTemplateS3Uri'] = args['ui_template']
+    request['HumanTaskConfig']['TaskTitle'] = args['title']
+    request['HumanTaskConfig']['TaskDescription'] = args['description']
+    request['HumanTaskConfig']['NumberOfHumanWorkersPerDataObject'] = args['num_workers_per_object']
+    request['HumanTaskConfig']['TaskTimeLimitInSeconds'] = args['time_limit']
+
+    if args['task_availibility']:
+        request['HumanTaskConfig']['TaskAvailabilityLifetimeInSeconds'] = args['task_availibility']
+    else:
+        request['HumanTaskConfig'].pop('TaskAvailabilityLifetimeInSeconds')
+
+    if args['max_concurrent_tasks']:
+        request['HumanTaskConfig']['MaxConcurrentTaskCount'] = args['max_concurrent_tasks']
+    else:
+        request['HumanTaskConfig'].pop('MaxConcurrentTaskCount')
+
+    if args['task_keywords']:
+    	for word in [n.strip() for n in args['task_keywords'].split(',')]:
+            request['HumanTaskConfig']['TaskKeywords'].append(word)
+    else:
+	    request['HumanTaskConfig'].pop('TaskKeywords')
+
+    ### Update worker configurations
+    if args['worker_type'].lower() == 'public':
+        if args['no_adult_content']:
+            request['InputConfig']['DataAttributes']['ContentClassifiers'].append('FreeOfAdultContent')
+        if args['no_ppi']:
+            request['InputConfig']['DataAttributes']['ContentClassifiers'].append('FreeOfPersonallyIdentifiableInformation')
+
+        request['HumanTaskConfig']['WorkteamArn'] = 'arn:aws:sagemaker:{}:394669845002:workteam/public-crowd/default'.format(args['region'])
+
+        dollars = int(args['workforce_task_price'])
+        cents = int(100 * (args['workforce_task_price'] - dollars))
+        tenth_of_cents = int((args['workforce_task_price'] * 1000) - (dollars * 1000) - (cents * 10))
+        request['HumanTaskConfig']['PublicWorkforceTaskPrice']['AmountInUsd']['Dollars'] = dollars
+        request['HumanTaskConfig']['PublicWorkforceTaskPrice']['AmountInUsd']['Cents'] = cents
+        request['HumanTaskConfig']['PublicWorkforceTaskPrice']['AmountInUsd']['TenthFractionsOfACent'] = tenth_of_cents
+    else:
+        request['InputConfig'].pop('DataAttributes')
+        request['HumanTaskConfig']['WorkteamArn'] = args['workteam_arn']
+        request['HumanTaskConfig'].pop('PublicWorkforceTaskPrice')
+
+    for key, val in args['tags'].items():
+        request['Tags'].append({'Key': key, 'Value': val})
+
+    return request
+
+
+def create_labeling_job(client, args):
+    """Create a SageMaker Ground Truth job"""
+    request = create_labeling_job_request(args)
+    try:
+        client.create_labeling_job(**request)
```

Comment:
does this not return any status?

Reply:
it just returns the ARN which isnt useful to us